releasenotes-builder/src/main/java/com/github/checkstyle/CliOptions.java

```diff
@@ -804,6 +816,17 @@ public Builder setSfRssProperties(String rssProps) {
             return this;
         }
 
+        /**
+         * Specify pom version.
+         *
+         * @param version pom version.
+         * @return Builder Object
+         */
+        public Builder setValidateVersion(String version) {
+            validateVersion = version;
```

Comment:
validate version should be boolean.
as we take version from another argument.

Reply:
Changes Done